### ISSUE TITLE
text tracks not supported

### ISSUE THREAD

**Issue by mikedo** [cta-wave/Media-Capability-APIs#7]
This API only supports video and audio. There is no support for text tracks such as TTML (IMSC1), WebVTT, etc.  I posted a note to the W3C Timed Text WG so maybe that will get corrected.

**Comment by haudiobe** [comment]
Proposal: Ask W3C about this.

**Comment by mikedo** [comment]
Yes. This post here was more FYI.  See above notice to W3C TTWG which convenes tomorrow.

See also: https://github.com/w3c/media-capabilities/issues/157 

I'll keep you posted.

**Comment by haudiobe** [comment]
@mikedo please formulate a request if needed.
